### DISCUSSION TITLE
Add a get_chat_members function

### DISCUSSION THREAD

**Discussion by soodi592** [kianahmadian/python-bale-bot#158] [Q&A] [answered]
There isnt this function in the library, but we need it for group managing bots, please add it soon.

**Comment by kian-ahmadian** [comment, marked as answer]
درود بر شما عزیز
این متد به همراه متد های جدید بله و مجموعه ای از تغییرات دیگر، به زودی اضافه خواهند شد.
از اطلاع رسانی شما متشکریم

**Comment by kian-ahmadian** [comment]
اصلاح: این متد در بله موجود نیست!
